test/search.cc

```diff
@@ -207,6 +207,75 @@ void search(valhalla::baldr::Location location, size_t result_count, int reachab
       throw std::logic_error("Wrong reachability");
 }
 
+
+
+
+void search_with_edge_headings(valhalla::baldr::Location location,
```

Comment:
it seems like you might not be done with this testing yet? i wouldnt duplicate this function, instead id update the existing tests to set the right heading on the expected values. the fake graph that is diagramed at the top of this file shows that each edge should have a pretty clear heading. like 0,90,180,270 and then we have also 45 and 225 approximately. You'll want to update the `shares` function to be sensitive to that, it is what we use to do these checks

Reply:
Yep, In the middle of testing.  Got it!　By the way where is the shares function?